<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgIMportTable</class>
 <widget class="QDialog" name="DlgIMportTable">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Data Into Table ...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qfrdrtable.qrc">
    <normaloff>:/table/table_import_large.png</normaloff>:/table/table_import_large.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnPivot">
       <property name="text">
        <string>Pivot/Rotate Table</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPasteAgain">
       <property name="text">
        <string>&amp;Paste Again</string>
       </property>
       <property name="icon">
        <iconset resource="qfrdrtable.qrc">
         <normaloff>:/table/paste.png</normaloff>:/table/paste.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+V</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>import where:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbImportWhere">
       <item>
        <property name="text">
         <string>append to table</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>overwrite table, starting top-left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>insert at cursor position</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>import what:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radImportAll">
       <property name="text">
        <string>import whole table</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radImportSelected">
       <property name="text">
        <string>import selected cells only</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chkImportExressions">
       <property name="text">
        <string>import cell/column expressions (if unchecked, all expressions will be deleted before importing the dataset)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>cmbImportWhere</tabstop>
  <tabstop>radImportAll</tabstop>
  <tabstop>radImportSelected</tabstop>
  <tabstop>chkImportExressions</tabstop>
  <tabstop>btnPivot</tabstop>
  <tabstop>btnPasteAgain</tabstop>
 </tabstops>
 <resources>
  <include location="qfrdrtable.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgIMportTable</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgIMportTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
